tests/steps/other_v2_steps.py

```diff
@@ -97,6 +98,32 @@ def read_program(context, path):
     return read_program_binary(path)
 
 
+# Send transactions to progress block numbers on dev mode.
+def burn_algo_transactions(context, num_txns=1):
+    sp = context.app_acl.suggested_params()
+    for _ in range(num_txns):
+        payment = transaction.PaymentTxn(
+            context.accounts[0],
+            sp,
+            constants.ZERO_ADDRESS,
+            random.randint(100000, 900000),
```

Comment:
@algochoi Optionally, I propose configuring the upper bound proportional to the funding amount.  Intent is to avoid generating a value that's too large if/when the funding amount changes.

Here's the Java SDK analog:  https://github.com/algorand/java-algorand-sdk/pull/350/commits/3390a6d687c5e57c349064de5014afc4f34a3b2d.

Feel welcomed to resolve as is.

Reply:
Good point - Added proportional funding amounts in latest commit https://github.com/algorand/py-algorand-sdk/pull/360/commits/3f110572a023ddadaffd0988b02a48a16721c77c